packages/@aws-cdk/assertions/lib/assertions.ts

```diff
@@ -0,0 +1,83 @@
+import { Stack, Stage } from '@aws-cdk/core';
+import * as assert from './vendored/assert';
+
+/**
+ * Suite of assertions that can be run on a CDK stack.
+ * Typically used, as part of unit tests, to validate that the rendered
+ * CloudFormation template has expected resources and properties.
+ */
+export class TemplateAssertions {
```

Comment:
Slight preference for `Assertions` here (shorter name, easier to read and type). But I guess potato potato.